arch-lifecycle/extensions/source/Arch.Lifecycle.Extensions/Transforms/Metadata.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+	<!--
```

Comment:
For now I didn't do any transformation to the generated bindings.

If there's something in particular you want me to look at - please let me know.

If you want me to clean up the template comments, I can do that.